unicodetools/data/ucd/dev/PropList.txt

```diff
@@ -1260,6 +1260,53 @@ E0020..E007F  ; Other_Grapheme_Extend # Cf  [96] TAG SPACE..CANCEL TAG
 
 # ================================================
 
+00B2..00B3    ; ID_Compat_Math_Continue # No    [2] SUPERSCRIPT TWO..SUPERSCRIPT THREE
```

Comment:
@eggrobin Did you run the UCD generator over the files, or is this a purely manual edit?

Reply:
All manual.